### ISSUE TITLE
Cannot start multiple Bandits as a Cowboy drop-in

### ISSUE THREAD

**Issue by adamu** [mtrudel/bandit#75]
I'm trying to replace Cowboy with Bandit. I have some test code that does something like this:

```elixir
{:ok, _} = start_supervised({Plug.Cowboy, scheme: :http, plug: Foo, options: [port: 4040]})
{:ok, _} = start_supervised({Plug.Cowboy, scheme: :http, plug: Bar, options: [port: 4041]})
```

However, replacing `Plug.Cowboy` with `Bandit` results in:

> ** (MatchError) no match of right hand side value: {:error, {:duplicate_child_name, Bandit}}

This seems to be because [Bandit uses `Bandit` as the child spec ID](https://github.com/mtrudel/bandit/blob/84c1f9cfc84cd48206abf9896fc47bcc8d43496e/lib/bandit.ex#L144-L146), but [`plug_cowboy` generates the `id` from the `plug` and `scheme`](https://github.com/elixir-plug/plug_cowboy/blob/a20571d1ab0a57fca902b92c45ed58ee82c55a09/lib/plug/cowboy.ex#L399-L401).

This can be worked around by specifying a custom ID, however as Bandit is advertised as a drop-in replacement for Cowboy, generating a unique `id` would allow multiple servers to be supervised using the `{Bandit, opts}` child spec format.

**Comment by mtrudel** [comment]
Yep, something like that makes sense. I'll get this integrated straight away. Thanks for the report!

**Comment by mtrudel** [comment]
Fix is merged into `main`. I'm waiting on a couple of other issues to be closed off before cutting another release, but this will be included in the next one (if it's a blocker for you I'm happy to cut one now). 

Thanks for the report!

**Comment by adamu** [comment]
Not a blocker, thanks for the super-fast turnaround!
